src/materials/mineclonia.lua

```diff
@@ -0,0 +1,81 @@
+--note this file handles mineclonia, mineclone2, and its rename voxellibre
```

Comment:
It's `voxelibre` afaik :-P

Reply:
https://github.com/mt-mods/xcompat/pull/22/commits/4dc905ae5d5ee62fb6d974246e67376257e0130f